test/unittests/api_test.cpp

```diff
@@ -100,9 +100,9 @@ TEST(api, resolve_imported_functions)
         *module, external_functions, {}, {}, std::vector<ExternalGlobal>(external_globals));
 
     EXPECT_THAT(execute(*instance, 0, {}), Result(0));
-    EXPECT_THAT(execute(*instance, 1, {Value{0}}), Result(1));
-    EXPECT_THAT(execute(*instance, 2, {Value{0}}), Result(2));
-    EXPECT_THAT(execute(*instance, 3, {0, 0}), Result());
+    EXPECT_THAT(execute(*instance, 1, {0_u32}), Result(1));
+    EXPECT_THAT(execute(*instance, 2, {0_u32}), Result(2));
+    EXPECT_THAT(execute(*instance, 3, {0_u64, 0}), Result());
```

Comment:
```suggestion
    EXPECT_THAT(execute(*instance, 3, {0_u64, 0_u32}), Result());
```

Reply:
Can be, but I wanted to stick with minimal set of changes here.

Reply:
Not sure I understand this motivation.

What is the type of the second argument passed? Presumably it's i32, because test passed. But it's some implicit conversion involved, it might be platform-dependent in general case and, it just looks weird.

Reply:
Agree with @gumb0 here. Would be more consistent to use `0_u64, 0_u32`.